src/lib.rs

```diff
@@ -53,6 +56,12 @@ fn get_internal() -> usize {
     unsafe { kernel32::GetCurrentThreadId() as usize }
 }
 
+#[cfg(target_os = "redox")]
+#[inline]
+fn get_internal() -> usize {
+    syscall::getpid().unwrap()
```

Comment:
I am not familiar with Redox internals, but on Linux at least there is a distinction between the process id and the thread id. Could you please add a comment to explain why `getpid` is the right thing to call here? Is there even a distinction between processes and threads on Redox?